.gitignore

```diff
@@ -1,6 +1,5 @@
 *.idea
 node_modules
 *.log*
-/dist
```

Comment:
Why include dist?

Reply:
Whoa! This one's more than two years old.

That's really why -- we use `dist` as a simple mechanism to publish so we don't have to put a competing module up in npm. We hadn't seen activity in this repo and so that's where we stuck the compiled code.

I do not think it needs to be included for the main repo if the main repo is starting to get updates again.

Reply:
And to be clear, I think this is because the PR is continuing to pull in our latest commits. When we included `dist`, it was after we'd given up that we'd get this PR merged.

Reply:
Hahaha, I missed the ping before, and for some reason GH pinged me again just yesterday, hence the question :)
As I am not using the lib anymore myself, would you like to be added as a contributor to keep maintaining it?

Reply:
However, it is probably easier to just redirect people to your fork

Reply:
Sure! I am open to doing it either way. I'd like to maintain this -- we have security patches as well as some new features to add. Its a great library, thank you for it.

Would you like to do the Transfer Ownership function so GH does an automatic redirect? I can drop the supervillain-software-group repo if so so we can keep all the repo history in the main one.

Reply:
Happy to do so! Could you rename your fork to anything else?
<img width="789" alt="изображение" src="https://github.com/user-attachments/assets/db6006c2-9c4b-491a-8bea-2f5c3b0320c1">

Reply:
@aigoncharov done! Sorry for that. I deleted it and will bring my local commits in after transfer.